dashboard/app/jobs/concerns/ai_rubric_config.rb

```diff
@@ -139,11 +137,14 @@ def self.validate_learning_goals_for_rubric(rubric)
       raise "Missing AI config in S3 for lesson #{lesson_s3_name} learning goals: #{missing_learning_goals.inspect}"
     end
   end
-  private_class_method :validate_learning_goals_for_rubric
 
   def self.get_s3_learning_goals(lesson_s3_name)
     rubric_csv = read_file_from_s3(lesson_s3_name, 'standard_rubric.csv')
     rubric_rows = CSV.parse(rubric_csv, headers: true).map(&:to_h)
     rubric_rows.map {|row| row['Key Concept']}
   end
+
+  private_class_method :validate_ai_config_for_lesson
+  private_class_method :validate_learning_goals
+  private_class_method :validate_learning_goals_for_rubric
```

Comment:
it seems very strange to separate the public/private status from the method definition. perhaps these should be inline instead? that's probably easier than updating the new lint rule to recognize this situation.
```
private_class_method def self.get_s3_learning_goals(lesson_s3_name)
```